app/templates/components/dropdown-content.hbs

```diff
@@ -1,5 +1,6 @@
 <div
   class={{classnames
+    @class
```

Comment:
It fixes the bug on production - ember global nav search suggestions dropdown content doesn't have right styles applied.